### DISCUSSION TITLE
General comments about GOVERNANCE.md

### DISCUSSION THREAD

**Discussion by jclingan** [commonhaus/foundation#2] [Q&A] [answered]
- **Commonhaus Council**
  - _Ensure the vitality and sustainability of projects._ How are these "ensured" without violating its goal of "Project Autonomy"? How would the Council intervene?
  - _Bolster collaboration between projects._ What is the intent behind this bullet? What do projects under the "Commonhaus Foundation" need to collaborate about?
- **Composition and Membership**
  -  What is the expected length of a term (annual?) and what if a Council Member becomes inactive during their tenure?

**Comment by ebullient** [comment]
> What is the expected length of a term (annual?) and what if a Council Member becomes inactive during their tenure?

I thought I had the terms (yes, annual) spelled out, but it looks like I dropped it by accident. Trying to make these things reasonably complete while still being readable was an interesting exercise with a lot of iterations. Will fix.

> Bolster collaboration between projects

I have ideas, but this would be community driven. This could cover collaborating on openrewrite rules to make it easier to address interdependent upgrades. We may have more cross-project (opt-in) activities around bug hunting or improving CI practices or other optimizations. I have no idea, but I wanted to leave room for benefits to the project that come from being in a foundation.

> without violating its goal of "Project Autonomy"? How would the Council intervene?

I would like to work this out with potential member projects. It is an idea I have now, but we'll have to see about who is comfortable with what, and how we can make that work. The easy one is finding new maintainers when the originators are done: that's a facilitated transition that should be made easier because the foundation holds the IP. The harder one is the emergency fix and the maintainer is unavailable for (pick a reason). There are a lot of things to consider for this one. Non-trivial question, and I would want input from project owners to iron this one out.

**Reply by ebullient** [reply]
Many many updates to https://github.com/commonhaus/foundation-draft/blob/main/bylaws/3-cf-council.md

- CFC members (councilors) can be removed (as described in the above doc), e.g. if they stop participating.
- Project autonomy vs. intervention is still an area to be explored.

**Comment by ebullient** [comment]
> > What is the expected length of a term (annual?) and what if a Council Member becomes inactive during their tenure?
> 
> I thought I had the terms (yes, annual) spelled out, but it looks like I dropped it by accident. Trying to make these things reasonably complete while still being readable was an interesting exercise with a lot of iterations. Will fix.

I put terms for the council (1 year) and the chair (2 years).. but those are flexible and subject to revision with more opinions than mine. ;)

**Comment by DanHeidinga** [comment]
Jumping into this discussion with a similar question: do meetings require quorum?  How many members must be present for quorum to be met?

With lazy consensus voting, it's easy for inactive council members to appear to support mailing list decisions.  For meetings, would you require members to attend so many meetings a year or lose their voting rights until they've attended N meetings in a row?

**Reply by ebullient** [reply]
Those are good questions. I would want to decide the actual logistics with the first council. This was an outline of an idea for flavor.. 

But it is a good question, and lack of participation with volunteer-lead groups is a real thing.

**Reply by ebullient** [reply]
- Quorum for meetings: https://github.com/commonhaus/foundation-draft/blob/main/bylaws/3-cf-council.md#quorum
- Electronic participation has been clarified (especially with regard to florida state law), for the CFC to electronically vote on something, all have to respond. Otherwise, you have a meeting, and then Quorum rules apply (re: who shows up).

**Comment by kenfinnigan** [comment]
The only piece missing to resolve this is the tenure. Maybe he's ok to leave it open for now?

**Reply by ebullient** [reply, marked as answer]
I don't have a requirement for a term limit, but I do think we have to align elections and terms (to ensure staggered elections)... so if you have annual elections, but not everyone can be up for elections at the same time, that creates a 2-3 year term...
